pkgs/top-level/all-packages.nix

```diff
@@ -7012,7 +7012,9 @@ in
     profiledCompiler = false;
     inherit zip unzip zlib boehmgc gettext pkgconfig perl;
     inherit (gnome2) libart_lgpl;
-  });
+  }).overrideAttrs (oldAttrs: {
+    strictDeps = stdenv.isDarwin;
```

Comment:
IIRC gcc has strictDeps=true alread?
/cc @Ericson2314

Reply:
If it does, it didn't in November when I first authored this commit. Although this should definitely be `oldAttrs.strictDeps or stdenv.isDarwin`.

Reply:
You were right! We didn't need this, so I've dropped this change.